src/api/user/index.ts

```diff
@@ -45,7 +46,7 @@ export type GetUserInfoResult =
 
 export type UpdateUserInformationResponse =
 	| {
-			status: "OK" | "EMAIL_ALREADY_EXISTS_ERROR" | "PHONE_ALREADY_EXISTS_ERROR";
+			status: "OK" | "EMAIL_ALREADY_EXISTS_ERROR" | "PHONE_ALREADY_EXISTS_ERROR" | "EMAIL_UPDATE_FORBIDDEN";
```

Comment:
Do not add extra fields to API response types, also this extra type is not needed